### PR TITLE
Fix/scan index

### PR DESCRIPTION
**Pull request by mts299**
# Scope 

While making fan plots found a scan index 

**issue:**  Didn't make one as I just fixed it :p

## Approval

**Number of approvals:** 1

## Test

``` python
import pydarn
from datetime import datetime
import matplotlib.pyplot as plt 

cly_file = 'data/20150308.1400.03.cly.fitacf'
pyk_file = 'data/20150308.1401.00.pyk.fitacf'

pyk_data = pydarn.SuperDARNRead().read_dmap(pyk_file)
cly_data = pydarn.SuperDARNRead().read_dmap(cly_file)

#pydarn.Fan.plot_fan(cly_data, scan_index=datetime(2015, 3, 8, 14, 4),
#                    colorbar=False, fov_color='grey', line_color='blue',
#                    radar_label=True)

pydarn.Fan.plot_fan(pyk_data, scan_index=datetime(2015, 3, 8, 14, 4), 
                    colorbar_label='Velocity [m/s]', fov_color='grey',
                    line_color='red', radar_label=True)

#pydarn.Fan.plot_fov(66, datetime(2015, 3, 8, 15, 0), radar_label=True)
plt.show()
```
fan_3

If you try on `develop` it will not work and raise an error of incorrect date.

### CODE REVIEW

File: docs/user/fan.md [hunk 1]
```diff
@@ -35,15 +35,26 @@ SDarn_read = pydarn.SuperDARNRead(file)
 fitacf_data = SDarn_read.read_fitacf()
 
 ```
-With the FITACF data loaded as a list of dictionaries (`fitacf_data` variable in above example), you may now call the `plot_fan` method. Make sure you tell it which scan (numbered from first recorded scan in file, counting from 1) you want using `scan_index`:
+With the FITACF data loaded as a list of dictionaries (`fitacf_data` variable in above example), you may now call the `plot_fan` method. Make sure you tell it which scan (numbered from first recorded scan in file, counting from 1 or give it a `datetime` object for the scan a that time) you want using `scan_index`:
```
Comment (carleyjmartin) [inline docs/user/fan.md:38]:
```suggestion
With the FITACF data loaded as a list of dictionaries (`fitacf_data` variable in above example), you may now call the `plot_fan` method. Make sure you tell it which scan (numbered from first recorded scan in file, counting from 1 or give it a `datetime` object for the scan at that time) you want using `scan_index`:
```

File: docs/user/fan.md [hunk 2]
```diff
@@ -76,18 +87,27 @@ In addition to line-of-sight velocity, you can choose one of three other data pr
...
+| colorbar_label=(string)       | Label for the colour bar (requires colorbar to be true)                                                 |
+| boundary=(bool)               | Set false to not show the outline of the radar FOV (default: True)                                      |
+| fov_color=(string)            | Sets the fill in color for the fov plot (default: transparency)                                         |
+| line_color=(string)           | Sets the boundary line and radar location dot color (default: black)                                    |
+| alpha=(int)                   | Sets the transparency of the fill color (default: 0.5)                                                  |
+| radar_location=(bool)         | Places a dot in the plot representing the radar location (default: True)                                |
+| radar_label=(bool)            | Places the radar 3-letter abbreviation next to the radar location                                       |
+| Kwargs **                     | Axis Polar settings. See [polar axis](axis.py)                                                          |
```
Comment (carleyjmartin) [inline docs/user/fan.md:106]:
```suggestion
| kwargs **                     | Axis Polar settings. See [polar axis](axis.py)                                                          |
```

File: pydarn/plotting/fan.py [hunk 3]
```diff
@@ -153,23 +154,22 @@ def plot_fan(cls, dmap_data: List[dict], ax=None,
...
-                                                     ['time.yr', 'time.mo',
-                                                      'time.dy', 'time.hr',
-                                                      'time.mt', 'time.sc']])
-                                       for record in dmap_data
-                                       if record['scan'] == 1])
-            # find corresponding scan_index
-            matching_scan_index = np.argwhere(scan_datetimes ==
-                                              scan_index)[..., 0] + 1
+            scan_time = scan_index
+            scan_index = 1
```
Comment (carleyjmartin) [inline pydarn/plotting/fan.py:158]:
```suggestion
            scan_index = 0
```
The time is shifted by one minute atm

**Review by carleyjmartin — COMMENTED** [review]
Works great, there was just one issue with the scan_index indexing from 0 basically, and the other comments are tiny typos

File: docs/user/fan.md (hunk 2 above)
Comment (mts299) [reply docs/user/fan.md]: fixing on docs PR

**Pull request merged**